starknet_py/hash/class_hash_test.py

```diff
@@ -1,6 +1,10 @@
 # pylint: disable=line-too-long
 # fmt: off
 import pytest
+from starkware.starknet.core.os.class_hash import (
+    compute_class_hash as sw_compute_class_hash,
+)
```

Comment:
I think it's fine for now, but we should try to avoid adding new functionalities based on cairo-lang library, even when it comes to tests.